postgres-appliance/spilok8s.yaml

```diff
@@ -85,7 +85,8 @@ kind: Endpoints
 metadata:
   name: &cluster_name spilodemo
   labels:
-    app: *cluster_name
+    application: spilo
+    spilo-cluster: *cluster_name
```

Comment:
Not consistent with what? camelCase is used for kubernetes objects and entities, we are using dashes for our objects (spilo-cluster, spilo-role).
And `cluster_name` is a very special case, it is used and the name for a yaml anchor and alias.